PX.Survey.Ext/PX.Survey.Ext.csproj

```diff
@@ -4,25 +4,25 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="CommonServiceLocator">
-      <HintPath>..\..\..\..\Bin\CommonServiceLocator.dll</HintPath>
```

Comment:
If this is a local path, I would suggest we leave it out of the commit to keep it cleaner.